### ISSUE TITLE
Aether blocks replaced with blocks from other mods

### ISSUE THREAD

**Issue by George0079** [gildedgames/aether-issues#717]
It's in the title. So far, I've only seen blocks from Applied Energistics 2 and Extreme Reactors, but I uninstalled the mod ASAP when I noticed the problem.

A simple ID change would do. Maybe add a few zeroes.

**Comment by LordMZTE** [comment]
i had the same issue but i cant get back my inventory because the only way to get back to the overworld i could see is by using a command help pls

**Comment by msticninja** [comment]
Yeah, this shouldn't be happening with Forge, and it's been a year...
![2019-03-06_18 32 08](https://user-images.githubusercontent.com/1581853/53923502-3140ef80-403e-11e9-9fe5-76960c7aaa6c.png)
